src/applications/disability-benefits/2346/Components/ConfirmAddress.jsx

```diff
@@ -0,0 +1,73 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getVeteranInformationData } from '../actions';
+
+class ConfirmAddress extends Component {
+  static propTypes = {
+    addressLine1: PropTypes.string.isRequired,
+    addressLine2: PropTypes.string.isRequired,
+    city: PropTypes.string.isRequired,
+    state: PropTypes.string.isRequired,
+    country: PropTypes.string.isRequired,
+    zip: PropTypes.string.isRequired,
+    email: PropTypes.string.isRequired,
+    getVeteranInformationData: PropTypes.func.isRequired,
+  };
+  static defaultProps = {
+    addressLine1: '',
+    addressLine2: '',
+    city: '',
+    state: '',
+    country: '',
+    zip: '',
+    email: '',
+    getVeteranInformationData: () => {},
+  };
+  componentDidMount() {
+    this.props.getVeteranInformationData();
+  }
+
+  render() {
+    const { addressLine1, addressLine2, city, state, zip, email } = this.props;
+
+    return (
+      <div>
+        <h1>Confirm your address</h1>
```

Comment:
This should be an `h2`; only one `h1` is allowed per page and the form title is an `h1`

Reply:
Done